docs/components/ComponentsView.jsx

```diff
@@ -367,6 +369,12 @@ ComponentsView.componentsToDisplay = [
     componentName: "WizardLayout",
     componentImgAlt: "A WizardLayout component",
   },
+  {
+    componentLink: "messaging-attachment",
+    componentImg: "messaging-attachment.png",
+    componentName: "MessagingAttachment",
+    componentImgAlt: "A MessagingAttachment component",
+  },
```

Comment:
Even though it's not strictly necessary since you added the sort up above, might still be nice to slot this in alphabetically? Just a nit, not blocking.

Reply:
I updated it to sort alphabetically, so it doesn't matter where it is in the list https://github.com/Clever/components/pull/655/files/e0cf0ccd55f088b01f105721881f0b1bb29201b4#diff-bfd693f512631919c327aa9c7211594ea3ac21bd5c114f4bb8e729ff7b2310d5R14

Reply:
Yeah, I just noticed the list was already alphabetical (I think?) except for this addition, so I couldn't help pointing it out - the urge to alphabetize is so strong. 😅